MANIFEST.in

```diff
@@ -4,19 +4,32 @@ include SYMLINK_CACHE.json
 include requirements.txt
 include shippable.yml
 include bin/ansible-test
+recursive-include docs *
+exclude docs/docsite/rst_warnings
+recursive-exclude docs/docsite/_build *
+recursive-exclude docs/docsite/extensions *.pyc *.pyo
 include examples/hosts
 include examples/ansible.cfg
 include examples/scripts/ConfigureRemotingForAnsible.ps1
 include examples/scripts/upgrade_to_ps3.ps1
-recursive-include lib/ansible/executor/powershell *
-recursive-include lib/ansible/module_utils/csharp *
-recursive-include lib/ansible/module_utils/powershell *
-recursive-include lib/ansible/modules *
-recursive-include lib/ansible/galaxy/data *
-recursive-include docs *
+recursive-include lib/ansible/executor/powershell *.ps1
+recursive-include lib/ansible/module_utils/csharp *.cs
+recursive-include lib/ansible/module_utils/powershell *.psm1
+recursive-include lib/ansible/modules/windows *.ps1
+recursive-include lib/ansible/galaxy/data *.yml *.j2 README.md ansible.cfg inventory .git_keep
+recursive-include lib/ansible/config/ *.yml
 recursive-include licenses *
 recursive-include packaging *
-recursive-include test *
+recursive-include test/cache .keep
+recursive-include test/integration *
+recursive-include test/legacy *
+recursive-exclude test/lib/ansible_test *.pyc *.pyo *.bak *.orig *~ *.swp
```

Comment:
Why do we need excludes here but not for `test/sanity` or `test/units`?

Reply:
If it's safe, I can target it at a different level but I wasn't sure of that.  sanity is probably okay.  units, there's a small chance of someone wanting to drop a data file in that uses those extensions.  integration, I think I would definitely avoid.

Things like PluginLoader tests might want to use a .pyo or .pyc file as part of a test case, for instance.

Reply:
mattclay and I discussed this and slimmed the MANIFEST file down to 6 directories with ```*``` wildcards.  Decided an eventual goal of removing all the wildcards would be better than blacklisting extensions.  Removed the extension blacklist in the latest version.